app/lib/submission_builder/ty2022/states/az/documents/financial_transaction.rb

```diff
@@ -0,0 +1,25 @@
+module SubmissionBuilder
+  module Ty2022
+    module States
+      module Az
+        module Documents
+          class FinancialTransaction < SubmissionBuilder::Document
+            include SubmissionBuilder::FormattingMethods
+
+            def document
+              build_xml_doc("efile:FinancialTransaction") do |xml|
```

Comment:
<img width="743" alt="Screenshot 2023-11-20 at 4 15 19 PM" src="https://github.com/codeforamerica/vita-min/assets/49880002/0e6582b5-5884-4f02-a795-bc9e9dbbee83">
seeing this error when I try to include financial transaction node for AZ